src/OpenTelemetry.Exporter.ZPages/ZPagesExporterStatsHttpServer.cs

```diff
@@ -121,9 +121,13 @@ private void WorkerThread()
                         using (var writer = new StreamWriter(output))
                         {
                             writer.WriteLine("Span Count : " + this.spanProcessor.GetSpanCount());
+                            writer.WriteLine("Error Count : " + this.spanProcessor.GetErrorCount());
+                            writer.WriteLine("Span Name\tStatus\t \tStart Timestamp\t \t \tEnd Timestamp\t \t \tLatency");
                             foreach (var spanData in this.spanProcessor.GetSpanDataBatch())
                             {
-                                writer.WriteLine("Span Name : " + spanData.Name + "\t" + "Status : " + spanData.Status.CanonicalCode);
+                                long latency = spanData.EndTimestamp.ToUnixTimeMilliseconds() > spanData.StartTimestamp.ToUnixTimeMilliseconds() ? spanData.EndTimestamp.ToUnixTimeMilliseconds() - spanData.StartTimestamp.ToUnixTimeMilliseconds() : 0;
```

Comment:
you don't have to hide the negative values. Those will indicate an issue with the data collection that will be easier to discover with zPages